frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -190,6 +188,7 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
       volumes,
       volumeMounts,
       envFrom,
+      connections,
```

Comment:
Should this be the projectConnections? need to look into what projectdetailscontext returns